src/transformers/cache_utils.py

```diff
@@ -950,6 +951,8 @@ def _dequantize(self, qtensor):
 
 class SinkCache(Cache):
     """
+    Deprecated.
```

Comment:
`SinkCache` has been broken on some edge cases for over a year, the issues are non-trivial to fix, and it is no longer relevant -- we can achieve a similar effect with a few other flags. See deprecation warning below.